File: Tareas/10/Código JAVA/tarea10.java
/*	
	Nombre: Leslie Estefanía Pérez Cerda
	Matrícula: 1737535
	Hora: Jueves V4-V6
*/

import javax.swing.JOptionPane;
public class tarea10{
	public static void main (String[] args){
		String entrada;
		Double x,a;
		String[] options={"1","2","3","4","Salir"};
		int v = JOptionPane.showOptionDialog(null,"Convertir de Pesos a ...\n 1.- Dolares\n 2.- Euros\n 3.- Yenes\n 4.- Dolares de Hong Kong","Monedas",JOptionPane.YES_NO_CANCEL_OPTION,JOptionPane.QUESTION_MESSAGE,null,options,options[0]);

		switch(v){
			case 0:
				entrada = JOptionPane.showInputDialog("Ingrese la cantidad a convertir: ");
				x = Double.parseDouble(entrada);
				a = x*0.05265;
				JOptionPane.showMessageDialog(null,+x+" Pesos equivalen a "+a+" Dolares\n\nTasa de cambio:\n1 MXP = 0.05265 USD");
				break;
			case 1:
				entrada = JOptionPane.showInputDialog("Ingrese la cantidad a convertir: ");
				x = Double.parseDouble(entrada);
				a = x*0.04487;
				JOptionPane.showMessageDialog(null,+x+" Pesos equivalen a "+a+" Euros\n\nTasa de cambio:\n1 MXP = 0.04487 EUR");
				break;
			case 2:
				entrada = JOptionPane.showInputDialog("Ingrese la cantidad a convertir: ");
				x = Double.parseDouble(entrada);
				a = x*5.91789;
				JOptionPane.showMessageDialog(null,+x+" Pesos equivalen a "+a+" Yenes\n\nTasa de cambio:\n1 MXP = 5.91789 JPY");
				break;
			case 3:
				entrada = JOptionPane.showInputDialog("Ingrese la cantidad a convertir: ");
				x = Double.parseDouble(entrada);
				a = x*0.40984;
				JOptionPane.showMessageDialog(null,+x+" Pesos equivalen a "+a+" Dolares de Hong Kong\n\nTasa de cambio:\n1 MXP = 0.40984 HKD");
				break;
			default:
			break;
		}//Fin de switch
	} //Fin de función main
} //Fin de clase tarea8

File: Tareas/10/Código RUBY/tarea10.rb
=begin
	Nombre: Leslie Estefanía Pérez Cerda
	Matrícula: 1737535
	Hora: Jueves V4-V6
=end

begin
	puts "\e[H\e[2J"
	print("\n\t\tMenu principal\n")
	print("\t\t==============\n")
	print("\tConvertir de Pesos a . . .\n")
	print("\t1.- Dolares\n")
	print("\t2.- Euros\n")
	print("\t3.- Yenes\n")
	print("\t4.- Dolares de Hong Kong\n")
	print("\t5.- Salir\n")
	print("\nSelecciona una opción --> ")
	n=gets().to_i

	case n
		when 1
			print("\nIngrese la cantidad a convertir: ")
			a = gets().to_f
			b = a*0.05265;
			print("\t#{a} Pesos equivalen a #{b} Dolares")
			print("\n\n\tTasa de cambio:\n\t1 MXP = 0.05265 USD")
			print("\n\nPresione cualquier tecla para volver al menu")
			gets
		when 2
			print("\nIngrese la cantidad a convertir: ")
			a = gets().to_f
			b = a*0.04487;
			print("\t#{a} Pesos equivalen a #{b} Euros")
			print("\n\n\tTasa de cambio:\n\t1 MXP = 0.04487 EUR")
			print("\n\nPresione cualquier tecla para volver al menu")
			gets
		when 3
			print("nIngrese la cantidad a convertir: ")
			a = gets().to_f
			b = a*5.91789;
			print("\t#{a} Pesos equivalen a #{b} Yenes")
			print("\n\n\tTasa de cambio:\n\t1 MXP = 5.91789 JPY")
			print("\n\nPresione cualquier tecla para volver al menu")
			gets
		when 4
			print("\nIngrese la cantidad a convertir: ")
			a = gets().to_f
			b = a*0.40984;
			print("\t#{a} Pesos equivalen a #{b} Dolares de Hong Kong")
			print("\n\n\tTasa de cambio:\n\t1 MXP = 0.40984 HKD")
			print("\n\nPresione cualquier tecla para volver al menu")
			gets
		else
			puts "\e[H\e[2J"
	end
end while n != 5